package.json

```diff
@@ -49,12 +48,12 @@
     "src"
   ],
   "dependencies": {
-    "immer": "^1.10.5",
+    "immer": "^1.12.1",
     "redux": "^4.0.0",
-    "redux-devtools-extension": "^2.13.7",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
Updating this was the main one to remove the `ts-ignore` but since I was here 🤷‍♂️